lib/rules/html-xml-lang-mismatch.json

```
@@ -2,7 +2,7 @@
   "id": "html-xml-lang-mismatch",
   "selector": "html[lang][xml\\:lang]",
   "matches": "xml-lang-mismatch-matches",
-  "tags": ["cat.language", "wcag2a", "wcag311", "ACT"],
+  "tags": ["cat.language", "wcag2a", "wcag311", "ACT", "TTv5", "TT11.a"],
```

I'm not seeing this, are you sure?

```suggestion
  "tags": ["cat.language", "wcag2a", "wcag311", "ACT"],
```